e2e_example/lib/copy_builder.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+library build.example.copy_builder;
```

Comment:
Down with the tyranny of library tags!

For real though, you can omit these without consequence now.